packages/styled/types/tests-base.tsx

```diff
@@ -98,6 +99,22 @@ const Canvas1 = styled('canvas', {
 ;<Canvas0 id="id-should-be-filtered" />
 ;<Canvas1 />
 
+const styledWithForwardedExtraProp = styled('div', {
+  // this used to work, but now complains because 'priority' is not a prop of 'div'
```

Comment:
the comment is relevant as a reason for this PR - it will look really odd though in the code after this gets merged in

Reply:
Fixed